valet_routes/router.js

```diff
@@ -0,0 +1,13 @@
+const router = require('express').Router();
+const controllers = require('../account_controllers/controllers');
+require('dotenv').config();
+
+router.route('/valet/:id').get(controllers.valetInfo.getValetData);
+router
+  .route('/valet/:id/operation-hours')
+  .put(controllers.valetInfo.editOperationHours);
+router
+  .route('/valet/:id/spots')
+  .put(controllers.valetInfo.editGarageParkingSpots);
+
+module.exports = router;
```

Comment:
small semantics thing, but both valets and drivers are users, the only things diferentiating them is a boolean in their accounts table entry. depending on how we decide to persist that user on the client side will dictate how the client should render these. might be important down the line when that is all set up.. might stay up late to implement a solution to persist that data since i see this being an issue we might need to solve to make everyones lives easier...

Reply:
Yea I agree, but having a boolean to differentiate them should be fine for first pass